package.json

```diff
@@ -29,15 +29,15 @@
     "axios": "^0.27.2",
     "date-fns": "^2.28.0",
     "deepmerge": "^4.2.2",
-    "expo": "^43.0.0",
-    "expo-location": "^13.0.4",
+    "expo": "^47.0.0",
+    "expo-location": "^15.0.1",
     "i18next": "^21.6.14",
     "miragejs": "^0.1.43",
     "poland-public-holidays": "^3.1.0",
-    "react": "16.13.1",
+    "react": "18.1.0",
     "react-hook-form": "^6.15.5",
     "react-i18next": "^11.15.7",
-    "react-native": "^0.67.4",
+    "react-native": "^0.70.5",
```

Comment:
Let's not forget to bump "@types/react-native" from "0.67.4" to whatever is the newest.